src/coprocessor/codec/mysql/decimal.rs

```diff
@@ -1746,30 +1744,36 @@ macro_rules! write_word {
 }
 
 macro_rules! read_word {
```

Comment:
Could we write it as a function?

Reply:
no, in this case, `macro_rules` has a better performance than function.